server/api/impl/admin/list.go

```diff
@@ -4,28 +4,64 @@
 package admin
 
 import (
+	"strings"
+
 	"github.com/mattermost/mattermost-plugin-apps/apps"
-	"github.com/mattermost/mattermost-plugin-apps/server/utils/md"
 )
 
-func (adm *Admin) ListApps() ([]*apps.App, md.MD, error) {
-	apps := adm.store.App().GetAll()
-	if len(apps) == 0 {
-		return nil, "no apps installed", nil
-	}
+func (adm *Admin) GetApp(appID apps.AppID) (*apps.App, error) {
```

Comment:
s/`GetApp`/`GetInstalledApp` (to keep in line with `ListInstalledApps`) ?

Reply:
Are all of these apps `installed` though?

Reply:
Yes, "Apps" are locally installed and active (unless disabled, but that's not implemented yet), and "Manifests" are "known" - i.e. listed in the marketplace.json, or added locally with `/apps debug-add-manifest`.